code/game/machinery/doors/blast_door.dm

```diff
@@ -78,6 +78,7 @@
 // Parameters: None
 // Description: Opens the door. No checks are done inside this proc.
 /obj/machinery/door/blast/proc/force_open()
+	sleep(10)
```

Comment:
delay should come after the door closes on a mob,  but before setting the door's operating var to false, and needs to be longer